### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by worldbeater**
The first time I've seen Avalonia's readme, I was a bit confused about how to quickly get started with the framework. So this readme update includes a new paragraph named "Getting Started" where I've tried to point out how to create a simple "Hello world" application! Also made some changes in document structure to avoid large fullscreen images taking too much place

### CODE REVIEW

**Comment by JimSEOW** [comment]
> Avalonia is a multi-platform windowing toolkit - somewhat like WPF - that is intended to be multi-platform. It supports XAML, lookless controls and a flexible styling system, and runs on Windows using Direct2D and other operating systems using Skia and OS-specific windowing backend (GTK, Cocoa, etc).

Avalonia is a **WPF inspired cross platform XAML UI interface** supporting a wide range of OSs: Windows (.NET Framework, .NET Core,  Android, Linux (GTK) , MacOS and iOS.

**Comment by kekekeks** [comment]
@JimSEOW 
We *aren't* using WPF or UWP. There is separate library for integration with WPF, but that's it.

File: readme.md [hunk 1]
```diff
@@ -1,70 +1,49 @@
...
 
-<a href='https://www.youtube.com/watch?v=NJ9-hnmUbBM' target='_blank'>![](https://i.ytimg.com/vi/NJ9-hnmUbBM/hqdefault.jpg)<a/>
+## Getting Started
 
-## NuGet
+Avalonia [Visual Studio Extension](https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.AvaloniaforVisualStudio) contains project and control templates that will help you get started. After installing it, open "New Project" dialog in Visual Studio, choose "Avalonia" in "Visual C#" section and press OK (<a href="https://github.com/AvaloniaUI/Avalonia/raw/master/docs/tutorial/images/add-dialogs.png">screenshot</a>). Now you can write code and markup that will work on multiple platforms!
 
-Avalonia is delivered as a NuGet package.
-You can find the packages here: ([stable(ish)](https://www.nuget.org/packages/Avalonia/), [nightly](https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed))
+Avalonia is also delivered as a <b>NuGet</b> package. You can find the packages here: ([stable(ish)](https://www.nuget.org/packages/Avalonia/), [nightly](https://github.com/AvaloniaUI/Avalonia/wiki/Using-nightly-build-feed))
```
Comment (kekekeks) [inline readme.md]: "also"? That's the only way to get the package.

File: readme.md [hunk 2]
```diff
@@ -1,70 +1,49 @@
...
 
-You can install the package like this:
-`Install-Package Avalonia -Pre`
+Use this command in Package Manager console to install the package:
+```
+Install-Package Avalonia -Pre
```
Comment (kekekeks) [inline readme.md]: One probably needs `Avalonia.Desktop` package.

File: readme.md (hunk 1 above)
Comment (Worldbeater) [reply readme.md]: Nice remark! I tried to point out that one can use Visual Studio extension and one can install packages manually; but didn't succeed 😄

File: readme.md (hunk 2 above)
Comment (Worldbeater) [reply readme.md]: Added this info, need a review!

**Pull request merged**